dev/com.ibm.etools.maven.liberty.integration/src/com/ibm/etools/maven/liberty/integration/xml/internal/XMLDocumentBuilder.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * IBM Confidential
+ * OCO Source Materials
+ * (C) Copyright IBM Corp. 2017 All Rights Reserved
+ * The source code for this program is not published or otherwise
+ * divested of its trade secrets, irrespective of what has
+ * been deposited with the U.S. Copyright Office.
+ */
+
+package com.ibm.etools.maven.liberty.integration.xml.internal;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
+
+import com.ibm.etools.maven.liberty.integration.internal.Trace;
+
+public abstract class XMLDocumentBuilder {
```

Comment:
There is a common version of this class too.  Are they the same and can this one be removed?

Reply:
Thanks for catching that. I've removed it.